README.md

```diff
@@ -45,3 +45,14 @@ with:
   DEB_DISTRO: jammy
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 ```
+
+## Running manually
+
+You can run this action locally on Ubuntu 22.04 ("jammy") or Debian 12 ("bookworm") and newer.
+Run `prepare.sh` once to set up the system and/or adopt to your needs, it is advised to run it as a separate user to avoid polluting your home directory.
```

Comment:
@jspricke did you mean adopt here or adapt?

Reply:
adapt, thx.